cluster-scope/base/operators.coreos.com/operatorgroups/nexodus/operatorgroup.yaml

```diff
@@ -2,7 +2,9 @@
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
-  name: apex
+  name: nexodus
 spec:
   targetNamespaces:
-    - apex
+    - nexodus
+    - nexodus-qa
```

Comment:
do we need -qa and -monitoring namespace also listed as a a targetNamespaces here?

Reply:
I believe we will need to add it as it is added here.

Reply:
It was not added in the apex version of it, so I was not sure of it.